protostar/starknet/cheatable_syscall_handler.py

```diff
@@ -91,7 +95,7 @@ def _call_contract(
         request = self._read_and_validate_syscall_request(
             syscall_name=syscall_name, segments=segments, syscall_ptr=syscall_ptr
         )
-        calldata = segments.memory.get_range_as_ints(
+        calldata: CallData = segments.memory.get_range_as_ints(
```

Comment:
It isn't necessary but I just thought it makes the code clearer.

If it's a problem, I can remove this.

Reply:
It's not a big issue. A new type would make more sense to me if there would be a function or method that would expect a parameter with such type. Currently, it's used only to cast an already typed value. 

...

I think the `CallData` should be used by `CheatableCachedState::contract_calls`.

Reply:
> It's not a big issue. A new type would make more sense to me if there would be a function or method that would expect a parameter with such type. Currently, it's used only to cast an already typed value.

It's also used to improve readability. I will remove this as it seems to be problematic.

> I think the `CallData` should be used by `CheatableCachedState::contract_calls`.

IIRC you yourself came up with the idea of using `CairoOrPythonData`. I can change this to `CallData`, honestly, I don't mind, but it would be good to know if there is no confusion about this.

Reply:
[remove calldata type](https://github.com/software-mansion/protostar/pull/1060/commits/626ddc310d17bd7c4f3c93fecd601e02e583dea4)